OpenRA.Mods.Common/Projectiles/GravityBomb.cs

```diff
@@ -47,17 +48,18 @@ public class GravityBomb : IProjectile, ISync
 		readonly GravityBombInfo info;
 		readonly Animation anim;
 		readonly ProjectileArgs args;
+		readonly WVec acceleration;
 		[Sync] WVec velocity;
 		[Sync] WPos pos;
-		[Sync] WVec acceleration;
 
 		public GravityBomb(GravityBombInfo info, ProjectileArgs args)
 		{
 			this.info = info;
 			this.args = args;
 			pos = args.Source;
-			velocity = new WVec(WDist.Zero, WDist.Zero, -info.Speed);
-			acceleration = new WVec(WDist.Zero, WDist.Zero, info.Acceleration);
+			var facing = (args.PassiveTarget - pos).Yaw.Facing;
+			velocity = -info.Velocity.Rotate(WRot.FromFacing(facing));
```

Comment:
Yes, `args.Facing` is what I was thinking about as well. Let me remind you again, I closed #11624 because of the strong issue on targettyping/validating empty air - multi-stage projectiles themselves need no additional plumbing rework on their own, ask any of the three `Fire*Warhead`s in https://github.com/GraionDilach/OpenRA.Mods.AS/tree/master/Traits/Warheads.